lib/browser/api/web-contents.ts

```diff
@@ -436,14 +433,30 @@ const addReturnValueToEvent = (event) => {
 // Add JavaScript wrappers for WebContents class.
 WebContents.prototype._init = function () {
   // The navigation controller.
-  NavigationController.call(this, this);
+  const navigationController = new NavigationController(this);
+  this.loadURL = navigationController.loadURL.bind(navigationController);
```

Comment:
This is subtly different behaviour to what we had before.  Before we were assigning to the prototype, here we are actually creating instance properties.  Not sure if that's a problem but worth raising as a difference

Reply:
yep, the reason it is this way is that you can't call the constructor of a class without `new`.